recipes/chiscore/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "chiscore" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 954cc473c141e8ccdde46c7188550d5009383e43ef6252afbbcc9b01c4597a41
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pytest-runner
+    - python >3
+    - setuptools >=39.0.1
+    - pip
+  run:
+    - python >3
+    - chi2comb-py >=0.0.4
+    - pytest-doctestplus >=0.3.0
```

Comment:
Does this need to be a runtime requirement? Or can it just be [a testing one](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-requirements)?
```suggestion
```

Reply:
It is needed in runtime because I tests are builtin as well: ``chiscore.test()``.